Flick/Cells/ListTableViewCell.swift

```diff
@@ -86,6 +86,9 @@ class ListTableViewCell: UITableViewCell {
 
     @objc private func seeAllMedia() {
         // TODO: Implement see all media action
+//        let listViewController = ListViewController()
+//        self.navigationController?.pushViewController(listViewController, animated: true)
```

Comment:
is it purposely commented out?

Reply:
ya. I think later we're gonna want to go to lists from clicking the see all button